pull_request_template.md

```diff
@@ -19,9 +19,9 @@ If you have any questions or problems, please give a comment in the PR.
 ##### General
 - [ ] PR name is meaningful
 - [ ] Purpose of the PR is understood
-- [ ] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
 - [ ] Commit descriptions are understandable and well formatted
-- [ ] If backport: add original PR number and target branch at top of this file: **Backport of PR#000 to bareos-2x**
+- [ ] Separate commit for CHANGELOG.md changes (message: "update CHANGELOG.md" or "update CHANGELOG.md for PR #0000"). PR number is verified.
+- [ ] If backport: add original PR number and target branch at top of this file: **Backport of PR #0000 to bareos-2x**
```

Comment:
```suggestion
- [ ] check backport line
``` 
Add this to top of file: **Backport of PR #0000 to bareos-2x** (remove if no backport)